15_perms_lab/permslab.txt

```diff
@@ -1,15 +1,17 @@
 1. Using the file structure in the current directory, write the commands that perform the following:
 
   - change permission of file src/main.c to r--r--r-- using chmod letter notation
-
+chmod a=r src/main.c
   - change permission of file include/player.h to r-xrw--r- using chmod octal (numbers) notation
-
+chmod 564 src/player.c
   - change permission of all files in Doc folder (without changing the permissions of Doc folder itself) to rw-------
-
+chmod 777 Doc/*.*
   - change permission of "lib" folder and all files and subdirectories inside it to rwxrwxrwx
-
+chmod 777 l*/*.* or chmod -R 777 lib
 2. create a new directory named "test" and inside an empty file named "secret". Change permissions of the file "secret" to ---------. Can you delete the file secret? can you delete the folder test? why?
-
+I can delete the empty file: rm test/secret
+Cannot delete test, since it is a folder (Need to use rm -r)
```

Comment:
Not exactly. It is a folder and you do need `rm -r` but that's not enough. As `test` folder has no write permission, you can't delete the file `secret` that lies inside, and without it being deleted, the folder itself can't be deleted.